pkg/controller/druid/druid_controller.go

```diff
@@ -100,6 +101,22 @@ func (r *ReconcileDruid) Reconcile(request reconcile.Request) (reconcile.Result,
 	if err := deployDruidCluster(r.client, instance); err != nil {
 		return reconcile.Result{}, err
 	} else {
-		return reconcile.Result{RequeueAfter: time.Second * 10}, nil
+		return reconcile.Result{RequeueAfter: lookupReconcileTime()}, nil
+	}
+}
+
+// lookupReconcileTime shall return reconcile time if set as ENV
+// else defaults to 10s
+func lookupReconcileTime() time.Duration {
+	val, exists :=  os.LookupEnv("RECONCILE_TIME")
```

Comment:
nit: RECONCILE_WAIT_ON_ERROR ?


also, can we parse that on startup and fail the operator startup if env variable was provided but invalid?

Reply:
sure make sense
But why WAIT ON ERROR ? curious :thinking:

Reply:
I re-read this code a bit and realized that env is actually used for non-error case, `RECONCILE_WAIT` was the correct term, my mistake.

Reply:
ah ha i have been using that in my local :), will send the change and doc too. @himanshug